src/components/NcModal/NcModal.vue

```diff
@@ -564,6 +567,9 @@ export default {
 		this.clearFocusTrap()
 		this.$el.remove()
 	},
+	updated() {
+		this.showModal = this.$refs.mask.style.display !== 'none'
```

Comment:
This looks terribly wrong 🙈 But it does work, though.

Reply:
Thats needed because a `v-show` on the `NcModal` sets both, mask and container, `display: none`.
Closing the modal sets `showModal` to `false`, but there is no place where it is set to `true` again.

So we need to check on the updated hook (when `v-show` is changed) if the mask is currently displayed or not.

Reply:
I understand why you need to do it, it just doesn't look "vue-ish" to me 😉

Have a look at my approach in #3767.